src/vos/vos_common.c

```diff
@@ -303,21 +305,67 @@ vos_tx_end(struct vos_container *cont, struct dtx_handle *dth_in,
 	err = umem_tx_end(vos_cont2umm(cont), err);
 
 cancel:
+	if (dtx_is_valid_handle(dth_in)) {
+		dae = dth->dth_ent;
+		if (dae != NULL);
```

Comment:
(style)  trailing semicolon indicates no statements, indent implies otherwise
(style)  trailing statements should be on next line